Makefile.am

```diff
@@ -199,7 +199,7 @@ libglog_la_SOURCES = $(gloginclude_HEADERS) \
 nodist_libglog_la_SOURCES = $(nodist_gloginclude_HEADERS)
 
 libglog_la_CXXFLAGS = $(PTRHEAD_CFLAGS) $(GFLAGS_CFLAGS) $(MINGW_CFLAGS) \
-                      $(AM_CXXFLAGS) -DNDEBUG
+                      $(AM_CXXFLAGS) -DNDEBUG -fPIC
```

Comment:
This change should be done upstream, in google/glog.  It really isn't specific to our fork.  Once it has been merged upstream, ping me and I'll pull a new google/glog.